Production/govuk_ios/ViewControllers/ViewController.swift

```diff
@@ -2,4 +2,22 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    private let color: UIColor
+
+    init(color: UIColor,
+         tabTitle: String) {
+        self.color = color
+        super.init(nibName: nil, bundle: nil)
+        self.title = tabTitle
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = color
+    }
+
```

Comment:
Looks like we need to add those swiftlint rules we talked about :) Or maybe not if you changed your mind?